### ISSUE TITLE
Quota and Quota Used % are blank when exported to PDF

### ISSUE THREAD

**Issue by jrandRTCC** [KelvinTegelaar/CIPP#2607]
### Description

1.) Go to Email & Exchange - Reports- Mailbox Statistics
2.) Export Visible Columns as PDF
3.) Quota and Quota Used show in CIPP, but not in the report

![CIPP Bug](https://github.com/KelvinTegelaar/CIPP/assets/66025354/50da8aed-a823-414f-966d-a7b8973328bf)

![CIPP Bug2](https://github.com/KelvinTegelaar/CIPP/assets/66025354/a72c1b1d-d321-47e7-9f98-85046a5fd1d3)

### Environment data

```PowerShell
Non-sponsored instance
Front end version number: 5.9.3
Back end version number: 5.9.3
Tried Tenant Cache Clear: true
Tried Token Cache Clear: false
```

**Comment by jrandRTCC** [comment]
I confirm that I have checked the documentation thoroughly and believe this to be an actual bug.

**Comment by kris6673** [comment]
Fairly sure this is cause we calculate it in the frontend and not backend.
Moving it to the backend should be a fairly easy fix I think?
I'll give it a go when I have the time

**Comment by kris6673** [comment]
I would like to work on this please!

**Comment by KelvinTegelaar** [comment]
We dont want to do this in backend, as we're trying to decrease the amount of APIs and this is a direct graph call. I'll see how we can add it to frontend instead. :)

**Comment by kris6673** [comment]
Fixed in v6.0.0
